src/components/step/shared.ts

```diff
@@ -0,0 +1,8 @@
+
+export interface StepElementProps {
+    text: string;
+    textAlign?: any;
```

Comment:
3/5 of these properties aren't used or usable by StepNextButton, no?  I guess I would lean toward separating out NextButtonProps into a separate interface.  But these styles will actually "work" if they are applied separately, right...if someone was using the StepNextButton as a component on it's own and hardcoding the styles?  (So I guess seems fine to leave it...mostly do not want any props that do nothing since that would be confusing downstream for anyone using the component)

Reply:
Yes, it could be used that way to override next button styles in a different context. In this case I was mainly trying to keep the style conversions in one place for all step components.